include/aspect/postprocess/sealevel.h

```diff
@@ -0,0 +1,150 @@
+/*
+  Copyright (C) 2011 - 2019 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_postprocess_sealevel_h
+#define _aspect_postprocess_sealevel_h
```

Comment:
general question: Would you rather write this a sealevel equation, or sea level equation? If the second, then maybe the postprocessor should be called `sea_level` instead of `sealevel`? I don't know about the naming conventions for this, but compared to most of our other postprocessors I would favor `sea_level` (as file name and here, `SeaLevel` can remain the class name).

Reply:
Sea_level is better indeed! Because it's sea level, and not sealevel